modules/manage/pages/monitor/monitoring-n1ql-query.adoc

```diff
@@ -1292,11 +1297,11 @@ To get a list of all logged completed requests using the Admin REST API:
 curl -u Administrator:pword http://localhost:8093/admin/completed_requests
 ----
 
-To get a list of all logged completed requests using {sqlpp}, including the query plan:
+To get a list of all logged completed requests using {sqlpp}, including only successful requests:
 
 [source,sqlpp]
 ----
-SELECT *, meta().plan FROM system:completed_requests;
+SELECT * FROM system:completed_requests WHERE errorCount = 0;
```

Comment:
The query should be :

SELECT * FROM system:completed_requests WHERE state = "completed" AND errorCount = 0;

Reply:
Done